docker/functions.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/env bash
```

Comment:
should be /usr/bin/env under normal circumstances

also, alpine won't find bash.

Reply:
fixed. thank you fadon!